lib/browser.js

```diff
@@ -73,7 +73,7 @@ class Browser {
       await page.goto(url, options)
     }
     // watchdog
-    await page.waitForFunction('window.AsciidoctorPDF === undefined || window.AsciidoctorPDF.status === undefined || window.AsciidoctorPDF.status === "ready"', { timeout: this.renderingTimeout })
+    await page.waitForFunction('window.AsciidoctorPDF === undefined || window.AsciidoctorPDF.status === undefined || window.AsciidoctorPDF.status === "ready"')
```

Comment:
Both timeouts are useful.
This timeout will make sure that Paged.js (JavaScript) completes this task in a reasonable time.

The rendering timeout is the time taken by Paged.js to create paginated content from an HTML document.